$NetBSD: patch-src_baselib_Makefile,v 1.1 2013/06/11 14:56:06 jaapb Exp $

Add netstring-pcre package, no longer provided by netstring by default
--- src/baselib/Makefile.orig	2012-12-07 15:17:09.000000000 +0000
+++ src/baselib/Makefile
@@ -1,6 +1,6 @@
 include ../../Makefile.config
 
-LIBS     := -package lwt.unix,netstring,cryptokit,findlib,tyxml,lwt.syntax,${LWT_EXTRA_PACKAGE}
+LIBS     := -package lwt.unix,netstring,netstring-pcre,cryptokit,findlib,tyxml,lwt.syntax,${LWT_EXTRA_PACKAGE}
 OCAMLC   := $(OCAMLFIND) ocamlc${BYTEDBG} ${THREAD}
 OCAMLOPT := $(OCAMLFIND) ocamlopt ${OPTDBG} ${THREAD}
 OCAMLDOC := $(OCAMLFIND) ocamldoc
